File: IGNORE/neofetch.py
import os
import subprocess

print(""" ,--,                                                   
,---.'|                                                   
|   | :                                                   
:   : |     ,--,                                          
|   ' :   ,--.'|         ,---,          ,--,              
;   ; '   |  |,      ,-+-. /  |       ,'_ /|  ,--,  ,--,  
'   | |__ `--'_     ,--.'|'   |  .--. |  | :  |'. \/ .`|  
|   | :.'|,' ,'|   |   |  ,"' |,'_ /| :  . |  '  \/  / ;  
'   :    ;'  | |   |   | /  | ||  ' | |  . .   \  \.' /   
|   |  ./ |  | :   |   | |  | ||  | ' |  | |    \  ;  ;   
;   : ;   '  : |__ |   | |  |/ :  | : ;  ; |   / \  \  \  
|   ,/    |  | '.'||   | |--'  '  :  `--'   \./__;   ;  \ 
'---'     ;  :    ;|   |/      :  ,      .-./|   :/\  \ ; 
          |  ,   / '---'        `--`----'    `---'  `--`  
           ---`-'                                         
                                                         """)

print("Your system is running kernel 5.15.8")
print("Would you like to go back?")
print("Type yes to go back. Or type no to exit out of TermI")
type = input(">>>: ")

if type == 'yes':
    os.chdir('../')
    subprocess.call("python home.py", shell=True)

File: services/neofetch.cpp
#include <iostream>
#include <unistd.h>
using namespace std;

int main(){
 int choice;
 cout << "Your system is running linux!\n" << ends;
 cout << "More stuff will be coming soon!\n" << ends;
 cout << "[1] Yes.\n" << ends;
 cout << "[2] No exit.\n" << ends;
 cout << "Go back?\n" << ends;
 cout << ">>>: " << ends;
 cin >> choice;
 if(choice == 1){
  chdir("../");
  system("./home");
 }
 if(choice == 2){
 system("exit");
 } 
}
